nodogsplash/Makefile

```diff
@@ -49,6 +48,10 @@ define Package/nodogsplash/install
 	$(INSTALL_DIR) $(1)/etc/init.d
 	$(INSTALL_DIR) $(1)/etc/uci-defaults
 	$(INSTALL_DIR) $(1)/usr/lib/nodogsplash
+	$(CP) ./files/etc/config/nodogsplash $(1)/etc/config/
+	$(CP) ./files/etc/init.d/nodogsplash $(1)/etc/init.d/
+	$(CP) ./files/etc/uci-defaults/40_nodogsplash $(1)/etc/uci-defaults/
+	$(CP) ./files/usr/lib/nodogsplash/restart.sh $(1)/usr/lib/nodogsplash/
```

Comment:
This still does not look good. We are using simple CP. This needs to be reworked.

We should make it more clear, which will be more better to read this install section and it will be simple as it is.
1. Create folder, then install files to it and then create another folder and install there files, which are there.
The order if you do first binaries, then config files, does not matter. Important is that it is not lost. 

This is done for binary (a few lines above) and then you are creating three folders and it might be messy.

Anyway, I would suggest to use installation macros, which are listed here:
https://openwrt.org/docs/guide-developer/packages#file_installation_macros

E.g., INSTALL_CONF for config file, INSTALL_BIN for init config, etc.

This makes sure that it has permissions on the router, which we want and that it has not been tinkerered with.